spryker/nginx/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+do_spryker_directory_create() {
+  mkdir -p /app/data/DE/cache/Yves/twig
+  mkdir -p /app/data/DE/cache/Zed/twig
+  mkdir -p /app/data/DE/logs
+  mkdir -p /app/data/common
+}
+
+do_spryker_config_create() {
+  # create .pgpass in home directory for postgres client
+  echo "$DATABASE_HOST:*:*:$DATABASE_USER:$DATABASE_PASSWORD" > ~/.pgpass
```

Comment:
Erm.. probably not, as the install command is going to be executed by code owner (I guess), so probably I should use "as_code_owner" fn.